### PR TITLE
multimedia/MuseScore: Updated script, removed wrong dependency.

### PR DESCRIPTION
**Pull request by antonioleal**
multimedia/MuseScore: Updated script, removed wrong dependency.

### CODE REVIEW

File: multimedia/MuseScore/MuseScore.SlackBuild [hunk 1]
```diff
@@ -90,7 +90,8 @@ cmake .. \
     -DQMAKE=/usr/lib"${LIBDIRSUFFIX}"/qt5/bin/qmake \
     -DQT_QMAKE_EXECUTABLE=/usr/lib"${LIBDIRSUFFIX}"/qt5/bin/qmake
 
-make install \
+CPU=$(($(grep -c processor /proc/cpuinfo)/2))
+make -j$CPU install \
```
Comment (willysr) [inline multimedia/MuseScore/MuseScore.SlackBuild:94]: please use MAKEFLAGS environment variable for this
let's keep the script clean

**Comment by antonioleal** [comment]
Defined new MAKEFLAGS environment variable as per Willy observation.
Looks better now. Thanks.

**Comment by willysr** [comment]
MAKEFLAGS is something you can define outside SlackBuild file, so basically just remove the MAKEFLAGS and leave it as per SBo template

**Comment by willysr** [comment]
@sbo-bot: build multimedia/MuseScore

**Comment by willysr** [comment]
merged

**Pull request closed without merging**